cloud-formation/scripts/post-dev-stack-creation.sh

```diff
@@ -3,6 +3,8 @@
 source ./utils.sh
 source ./stack-name.sh
 
+export AWS_PROFILE=media-service
```

Comment:
@akash1810 We tried and no, since these scripts only define functions that call the awscli, they don't call the cli yet.